definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-bailiff-nonprod.json

```diff
@@ -4,7 +4,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "CertificateOfServiceDate",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "CRU"
+    "CRUD": "CRUD"
```

Comment:
these fields will be cleaned up and can be reused. We know from experience CRUD is the safest way.